src/app/_mermaid_common/libs/directives/leaflet-map.dir.js

```diff
@@ -70,6 +94,17 @@ angular.module('mermaid.libs').directive('leafletMap', [
             var center = defaultCenter;
             scope.maprecords.clearLayers();
             _.each(scope.records, function(rec) {
+              if (scope.mapopts.showPopup) {
+                var rec_geo_data = {
+                  id: rec.id,
+                  name: rec.name,
+                  project_id: scope.mapopts.project_id,
+                  reefexposures: rec.$$reefexposures.name,
```

Comment:
I think the keys should be singular, e.g. `reefexposure: rec.$$reefexposures.name`